install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1875,6 +1875,9 @@ ipam:
     # refill the bucket up to the burst size capacity.
     # @default -- `4.0`
     externalAPILimitQPS: ~
+    # Install ingress/egress routes through uplink on host for Pods when working with
+	  # delegated IPAM plugin.
+    installUplinkRoutesForDelegatedIPAM: false
```

Comment:
IMO this should be `ipam.installUplinkRoutesForDelegatedIPAM`, not `ipam.operator.installUplinkRoutesForDelegatedIPAM`, it's not a setting that applies to cilium-operator after all.

Reply:
@gandro yes, I intented to use `ipam.installUplinkRoutesForDelegatedIPAM` but placed to the wrong section. Will update later.